lib/services/ios-project-service.ts

```diff
@@ -844,6 +848,41 @@ We will now place an empty obsolete compatability white screen LauncScreen.xib f
 		return Promise.resolve();
 	}
 
+	public checkForChanges(changesInfo: IProjectChangesInfo, options: IProjectChangesOptions, projectData: IProjectData): void {
+		const nextCommandProvisionUUID = options.provision;
+
+		if (typeof nextCommandProvisionUUID === "string") {
```

Comment:
what if it's not a string? Shall we throw an error or log something to the user - otherwise he won't know something's wrong and we don't use the provided provision?